src/table/boot.rs

```diff
@@ -548,23 +548,22 @@ impl BootServices {
         }
     }
 
-    /// Load an EFI image from a buffer.
-    pub fn load_image_from_buffer(
+    /// Load an EFI image.
+    pub fn load_image(
         &self,
+        boot_policy: bool,
```

Comment:
Now that we are exposing this parameter in the API, could we model it as an enum? I'm thinking about the downstream users of this method, for whom it won't be very clear whether `boot_policy` should be `true` or `false`.